### ISSUE TITLE
Dev flow breaks on linux

### ISSUE THREAD

**Issue by smartcontracts** [0xOlias/ponder#78]
Hello! It seems the dev flow breaks on linux, at least with node >=14. Simple to reproduce, just create a fresh project and `yarn dev` (or `npm dev` or `pnpm dev`) and observe: 

```
TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
    at new NodeError (node:internal/errors:387:5)
    at watch (node:fs:2299:11)
    ...
```

[Probably related to this](https://stackoverflow.com/questions/61806341/how-to-fix-the-feature-watch-recursively-is-unavailable-on-the-current-platform). Recursive watch isn't available on linux so we get an error.

**Comment by 0xOlias** [comment]
@smartcontracts this should be fixed in `@ponder/core@0.0.27`. Was able to repro in CI, then fixed by migrating from a hacky `fs.watch` implementation to `chokidar`.

**Comment by smartcontracts** [comment]
Well that was fast
